CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.1 (2018-06-18)
+
+- Fix crash bug where backwash compared pset scores with cset scores.
```

Comment:
After my first few reads of this sentence, I was still struggling with the impression that the problem was that it was incorrect to compare a score from the pset with a score from the cset. To be clear, in this comment, do you mean "The code which compared scores crashed, because one of the scores was not converted to a number" ?